UserBackgrounds/index.tsx

```diff
@@ -176,15 +192,15 @@ export default class UserBackgrounds extends BasePlugin {
 
             return (
                 <div className={styles.container} onContextMenu={handleContextMenu}>
-                    <TextBadge color={Colors.BRAND_NEW_500} text={selection === 0 ? "USRBG" : "NATIVE"} className={styles.badge} />
+                    <TextBadge color={Colors.BRAND_NEW_500} text={selection === 0 ? "GUILD" : selection === 1 ? "NATIVE" : "USRBG"} className={styles.badge} />
                     {
                         (banner != null && user.banner != null) && (
                             <Button
                                 className={Utilities.className(styles.arrow, styles.left)}
                                 key="left"
                                 look={Button.Looks.BLANK}
                                 size={Button.Sizes.TINY}
-                                onClick={setSelection.bind(null, 0)}
+                                onClick={selection === 1 ? setSelection.bind(null, 0) : setSelection.bind(null, 1)}

```

Comment:
Better practice would be to do something like this instead: (same for the usage below)
```js
setSelection.bind(null, selection === 1 ? 0 : 1)
```